css/at-rules/supports.json

```diff
@@ -7,7 +7,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@supports",
           "spec_url": [
             "https://drafts.csswg.org/css-conditional-4/#at-supports-ext",
-            "https://drafts.csswg.org/css-conditional/#at-supports"
+            "https://drafts.csswg.org/css-conditional-3/#at-supports"
```

Comment:
This now looks like linking to a dated spec given we also link to level 4. Should we just omit the link to level 3?

Reply:
I think it’d be fine to drop it. But to be clear, this is one of those cases where the Level 4 version of the spec only defines extensions to the feature — rather than having a complete definition of the feature. The core definition for the actual feature is still in the Level 3 spec.

Reply:
Oh I see, well in that case, let's keep level 3.

/me is still getting used to leveled specs